gui/src/app/RunPanel/RunPanel.tsx

```diff
@@ -188,4 +188,31 @@ const LinearProgressWithLabel = (props: LinearProgressProps & { value: number })
     );
 }
 
+
+type DrawsDisplayProps = {
+    draws: number[][],
+    paramNames: string[]
+}
+
+const DrawsDisplay: FunctionComponent<DrawsDisplayProps> = ({ draws, paramNames }) => {
+    const means: { [k: string]: number } = {};
+
+    for (const [i, element] of paramNames.entries()) {
+        let sum = 0;
+        for (const draw of draws[i]) {
+            sum += draw;
+        }
+        means[element] = sum / draws[i].length;
+    }
```

Comment:
Other than my temptation to replace this with an inscrutable one-liner, this looks good on inspection. Will try kicking the tires in a sec.

Reply:
I'm assuming there's no valid condition where one chain would have 0 draws. (Or--at that point something has already blown up so we don't need to worry about the divide-by-zero, perhaps?)

Reply:
Sample has a precondition that the number of draws is at least 1, and a successful return means we have all the requested draws, so we don't need to worry here